### ISSUE TITLE
Core cumped when opening any video

### ISSUE THREAD

**Issue by kattjevfel** [easymodo/qimgv#259]
Opening a video file (mp4, webm) either directly or browsing a folder makes qimgv instantly crash.
```
katt@main ~ » qimgv /home/katt/Videos/YWMaGkw1KmslOJV_.mp4
terminate called after throwing an instance of 'std::runtime_error'
  what():  could not initialize OpenGL
[1]    12298 abort (core dumped)  qimgv /home/katt/Videos/YWMaGkw1KmslOJV_.mp4
```
Arch Linux
Linux 5.8.7-zen1-1-zen
mesa 20.1.7-1
qimgv-git v0.9.2.alpha.r0.g7b7f9c1-1
mpv-git 0.32.0_714_g5a4fc8684e-1

Not sure what other information could be useful, playing videos in mpv works fine and glxgears (OpenGL) works just fine.

```coredump
                Stack trace of thread 12298:
                #0  0x00007fef6976f615 raise (libc.so.6 + 0x3d615)
                #1  0x00007fef69758862 abort (libc.so.6 + 0x26862)
                #2  0x00007fef69af186a _ZN9__gnu_cxx27__verbose_terminate_handlerEv (libstdc++.so.6 + 0x9686a)
                #3  0x00007fef69afdd9a _ZN10__cxxabiv111__terminateEPFvvE (libstdc++.so.6 + 0xa2d9a)
                #4  0x00007fef69afde07 _ZSt9terminatev (libstdc++.so.6 + 0xa2e07)
                #5  0x00007fef69afe0ae __cxa_throw (libstdc++.so.6 + 0xa30ae)
                #6  0x00007fef5000a09f _ZN9MpvWidget12initializeGLEv (player_mpv.so + 0x809f)
                #7  0x00007fef6d9073f9 _ZN13QOpenGLWidget5eventEP6QEvent (libQt5Widgets.so.5 + 0x1ba3f9)
                #8  0x00007fef6d8a6702 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x159702)
                #9  0x00007fef6b25879a _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt5Core.so.5 + 0x2b179a)
                #10 0x00007fef6d8d96e8 n/a (libQt5Widgets.so.5 + 0x18c6e8)
                #11 0x00007fef6d8d96bd n/a (libQt5Widgets.so.5 + 0x18c6bd)
                #12 0x00007fef6d8e8f8a _ZN7QWidget9setParentEPS_6QFlagsIN2Qt10WindowTypeEE (libQt5Widgets.so.5 + 0x19bf8a)
                #13 0x00005599b1a2c7c4 _ZN20VideoPlayerInitProxy10initPlayerEv (qimgv + 0x1197c4)
                #14 0x00005599b1a2c0fa _ZN20VideoPlayerInitProxy4showEv (qimgv + 0x1190fa)
                #15 0x00005599b1a2d548 _ZN12ViewerWidget17enableVideoPlayerEv (qimgv + 0x11a548)
                #16 0x00005599b1a2e40b _ZN12ViewerWidget9showVideoE7QString (qimgv + 0x11b40b)
                #17 0x00005599b19db0c7 _ZN2MW8setVideoE7QString (qimgv + 0xc80c7)
                #18 0x00005599b1983792 _ZN4Core11guiSetImageESt10shared_ptrI5ImageE (qimgv + 0x70792)
                #19 0x00005599b19840a5 _ZN4Core16onModelItemReadyESt10shared_ptrI5ImageE (qimgv + 0x710a5)
                #20 0x00005599b1986b19 _ZN9QtPrivate11FunctorCallINS_11IndexesListIJLi0EEEENS_4ListIJSt10shared_ptrI5ImageEEEEvM4CoreFvS6_EE4callESA_PS8_PPv (qimgv + 0x73b19)
                #21 0x00007fef6b28fa06 n/a (libQt5Core.so.5 + 0x2e8a06)
                #22 0x00005599b1963716 _ZN14DirectoryModel9itemReadyESt10shared_ptrI5ImageE (qimgv + 0x50716)
                #23 0x00005599b199dbb8 _ZN14DirectoryModel4loadE7QStringb (qimgv + 0x8abb8)
                #24 0x00005599b197f854 _ZN4Core9loadIndexEibb (qimgv + 0x6c854)
                #25 0x00005599b198044f _ZN4Core8loadPathE7QString (qimgv + 0x6d44f)
                #26 0x00005599b195ee3b main (qimgv + 0x4be3b)
                #27 0x00007fef6975a152 __libc_start_main (libc.so.6 + 0x28152)
                #28 0x00005599b195f7ee _start (qimgv + 0x4c7ee)
                
                Stack trace of thread 12301:
                #0  0x00007fef6982746f __poll (libc.so.6 + 0xf546f)
                #1  0x00007fef68db3168 n/a (libglib-2.0.so.0 + 0xa0168)
                #2  0x00007fef68d64221 g_main_context_iteration (libglib-2.0.so.0 + 0x51221)
                #3  0x00007fef6b2b1311 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x30a311)
                #4  0x00007fef6b25711c _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2b011c)
                #5  0x00007fef6b072c32 _ZN7QThread4execEv (libQt5Core.so.5 + 0xcbc32)
                #6  0x00007fef654e4098 n/a (libQt5DBus.so.5 + 0x17098)
                #7  0x00007fef6b073e0f n/a (libQt5Core.so.5 + 0xcce0f)
                #8  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #9  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12302:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12299:
                #0  0x00007fef6982746f __poll (libc.so.6 + 0xf546f)
                #1  0x00007fef65a0963b n/a (libxcb.so.1 + 0xc63b)
                #2  0x00007fef65a0b37b xcb_wait_for_event (libxcb.so.1 + 0xe37b)
                #3  0x00007fef655fbc51 n/a (libQt5XcbQpa.so.5 + 0x5dc51)
                #4  0x00007fef6b073e0f n/a (libQt5Core.so.5 + 0xcce0f)
                #5  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #6  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12312:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12308:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12329:
                #0  0x00007fef6982cfcb __mprotect (libc.so.6 + 0xfafcb)
                #1  0x00007fef697bb217 sysmalloc (libc.so.6 + 0x89217)
                #2  0x00007fef697bcd6e _int_malloc (libc.so.6 + 0x8ad6e)
                #3  0x00007fef697bde51 malloc (libc.so.6 + 0x8be51)
                #4  0x00007fef6557c20d n/a (libfontconfig.so.1 + 0x2920d)
                #5  0x00007fef6557f2f8 n/a (libfontconfig.so.1 + 0x2c2f8)
                #6  0x00007fef68c5d9e8 n/a (libexpat.so.1 + 0xc9e8)
                #7  0x00007fef68c5bddd n/a (libexpat.so.1 + 0xaddd)
                #8  0x00007fef68c5f621 n/a (libexpat.so.1 + 0xe621)
                #9  0x00007fef68c618cc XML_ParseBuffer (libexpat.so.1 + 0x108cc)
                #10 0x00007fef6557d025 n/a (libfontconfig.so.1 + 0x2a025)
                #11 0x00007fef6557d658 n/a (libfontconfig.so.1 + 0x2a658)
                #12 0x00007fef6557d714 n/a (libfontconfig.so.1 + 0x2a714)
                #13 0x00007fef6557dea2 n/a (libfontconfig.so.1 + 0x2aea2)
                #14 0x00007fef68c5d9e8 n/a (libexpat.so.1 + 0xc9e8)
                #15 0x00007fef68c5bddd n/a (libexpat.so.1 + 0xaddd)
                #16 0x00007fef68c5f621 n/a (libexpat.so.1 + 0xe621)
                #17 0x00007fef68c618cc XML_ParseBuffer (libexpat.so.1 + 0x108cc)
                #18 0x00007fef6557d025 n/a (libfontconfig.so.1 + 0x2a025)
                #19 0x00007fef6557d658 n/a (libfontconfig.so.1 + 0x2a658)
                #20 0x00007fef40039c8b n/a (libass.so.9 + 0x24c8b)
                #21 0x00007fef4002172b n/a (libass.so.9 + 0xc72b)
                #22 0x00007fef40028a92 ass_set_fonts (libass.so.9 + 0x13a92)
                #23 0x00007fef402113a1 mp_ass_configure_fonts (libmpv.so.1 + 0xdb3a1)
                #24 0x00007fef402186ca create_ass_renderer (libmpv.so.1 + 0xe26ca)
                #25 0x00007fef40218d66 update_external (libmpv.so.1 + 0xe2d66)
                #26 0x00007fef401e5b4f cmd_osd_overlay (libmpv.so.1 + 0xafb4f)
                #27 0x00007fef401f1daa run_command (libmpv.so.1 + 0xbbdaa)
                #28 0x00007fef401e140c run_client_command (libmpv.so.1 + 0xab40c)
                #29 0x00007fef401e31ed mpv_command_node (libmpv.so.1 + 0xad1ed)
                #30 0x00007fef401fdee2 script_command_native (libmpv.so.1 + 0xc7ee2)
                #31 0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #32 0x00007fef206f8790 lua_pcall (libluajit-5.1.so.2 + 0x4e790)
                #33 0x00007fef401fc710 script_autofree_trampoline (libmpv.so.1 + 0xc6710)
                #34 0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #35 0x00007fef401fce9a load_scripts (libmpv.so.1 + 0xc6e9a)
                #36 0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #37 0x00007fef206f8790 lua_pcall (libluajit-5.1.so.2 + 0x4e790)
                #38 0x00007fef401fdcb6 run_lua (libmpv.so.1 + 0xc7cb6)
                #39 0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #40 0x00007fef206f87cc lua_cpcall (libluajit-5.1.so.2 + 0x4e7cc)
                #41 0x00007fef401fb3fe load_lua (libmpv.so.1 + 0xc53fe)
                #42 0x00007fef40205d87 run_script (libmpv.so.1 + 0xcfd87)
                #43 0x00007fef4020625b script_thread (libmpv.so.1 + 0xd025b)
                #44 0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #45 0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12309:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12311:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12315:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12307:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12322:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12305:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12320:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12328:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef401c902c mp_dispatch_lock (libmpv.so.1 + 0x9302c)
                #2  0x00007fef401e13dd lock_core (libmpv.so.1 + 0xab3dd)
                #3  0x00007fef401fe7ab script_commandv (libmpv.so.1 + 0xc87ab)
                #4  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #5  0x00007fef401fce9a load_scripts (libmpv.so.1 + 0xc6e9a)
                #6  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #7  0x00007fef206f8790 lua_pcall (libluajit-5.1.so.2 + 0x4e790)
                #8  0x00007fef401fdcb6 run_lua (libmpv.so.1 + 0xc7cb6)
                #9  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #10 0x00007fef206f87cc lua_cpcall (libluajit-5.1.so.2 + 0x4e7cc)
                #11 0x00007fef401fb3fe load_lua (libmpv.so.1 + 0xc53fe)
                #12 0x00007fef40205d87 run_script (libmpv.so.1 + 0xcfd87)
                #13 0x00007fef4020625b script_thread (libmpv.so.1 + 0xd025b)
                #14 0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #15 0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12314:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12306:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12313:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12317:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12327:
                #0  0x00007fef6971f9c8 pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf9c8)
                #1  0x00007fef401e0dc5 wait_wakeup (libmpv.so.1 + 0xaadc5)
                #2  0x00007fef401e2585 mpv_wait_event (libmpv.so.1 + 0xac585)
                #3  0x00007fef401fde1b script_raw_wait_event (libmpv.so.1 + 0xc7e1b)
                #4  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #5  0x00007fef206f8790 lua_pcall (libluajit-5.1.so.2 + 0x4e790)
                #6  0x00007fef401fc710 script_autofree_trampoline (libmpv.so.1 + 0xc6710)
                #7  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #8  0x00007fef401fce9a load_scripts (libmpv.so.1 + 0xc6e9a)
                #9  0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #10 0x00007fef206f8790 lua_pcall (libluajit-5.1.so.2 + 0x4e790)
                #11 0x00007fef401fdcb6 run_lua (libmpv.so.1 + 0xc7cb6)
                #12 0x00007fef206b2936 n/a (libluajit-5.1.so.2 + 0x8936)
                #13 0x00007fef206f87cc lua_cpcall (libluajit-5.1.so.2 + 0x4e7cc)
                #14 0x00007fef401fb3fe load_lua (libmpv.so.1 + 0xc53fe)
                #15 0x00007fef40205d87 run_script (libmpv.so.1 + 0xcfd87)
                #16 0x00007fef4020625b script_thread (libmpv.so.1 + 0xd025b)
                #17 0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #18 0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12319:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12310:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12318:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12324:
                #0  0x00007fef6982746f __poll (libc.so.6 + 0xf546f)
                #1  0x00005599b19cd7d4 poll (qimgv + 0xba7d4)
                #2  0x00007fef6b28fa06 n/a (libQt5Core.so.5 + 0x2e8a06)
                #3  0x00007fef6b071bab _ZN7QThread7startedENS_14QPrivateSignalE (libQt5Core.so.5 + 0xcabab)
                #4  0x00007fef6b073df5 n/a (libQt5Core.so.5 + 0xccdf5)
                #5  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #6  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12316:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12331:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12332:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12304:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12303:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12321:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12326:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef401c8d66 mp_dispatch_queue_process (libmpv.so.1 + 0x92d66)
                #2  0x00007fef4020209a mp_wait_events (libmpv.so.1 + 0xcc09a)
                #3  0x00007fef40204a12 mp_idle (libmpv.so.1 + 0xcea12)
                #4  0x00007fef401fb256 mp_play_files (libmpv.so.1 + 0xc5256)
                #5  0x00007fef401e2129 core_thread (libmpv.so.1 + 0xac129)
                #6  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #7  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 12323:
                #0  0x00007fef6971f6a2 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xf6a2)
                #1  0x00007fef5c686e3c n/a (radeonsi_dri.so + 0x4b0e3c)
                #2  0x00007fef5c685838 n/a (radeonsi_dri.so + 0x4af838)
                #3  0x00007fef697193e9 start_thread (libpthread.so.0 + 0x93e9)
                #4  0x00007fef69832293 __clone (libc.so.6 + 0x100293)
```

```gdb
Reading symbols from /usr/bin/qimgv...
Reading symbols from /usr/lib/debug/usr/bin/qimgv.debug...
[New LWP 12298]
[New LWP 12301]
[New LWP 12302]
[New LWP 12299]
[New LWP 12312]
[New LWP 12308]
[New LWP 12329]
[New LWP 12309]
[New LWP 12311]
[New LWP 12315]
[New LWP 12307]
[New LWP 12322]
[New LWP 12305]
[New LWP 12320]
[New LWP 12328]
[New LWP 12314]
[New LWP 12306]
[New LWP 12313]
[New LWP 12317]
[New LWP 12327]
[New LWP 12319]
[New LWP 12310]
[New LWP 12318]
[New LWP 12324]
[New LWP 12316]
[New LWP 12331]
[New LWP 12332]
[New LWP 12304]
[New LWP 12303]
[New LWP 12321]
[New LWP 12326]
[New LWP 12323]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
Core was generated by `qimgv /home/katt/Videos/YWMaGkw1KmslOJV_.mp4'.
Program terminated with signal SIGABRT, Aborted.
#0  0x00007fef6976f615 in raise () from /usr/lib/libc.so.6
[Current thread is 1 (Thread 0x7fef659e93c0 (LWP 12298))]
```

**Comment by easymodo** [comment]
can you try with mpv from arch repo?

**Comment by kattjevfel** [comment]
Works with the the one from the repo, but that one is obviously older, it's only a matter of time until that too catches up.
Either way, I ended up making `qimgv-light` without any extra features, since I don't really use them.

**Comment by nick-s-b** [comment]
Strange bug. I've tried to track it down but was lost. Dev branch is also crashing on any webm/mp4.

**Comment by easymodo** [comment]
should be fixed in dev branch

**Comment by nick-s-b** [comment]
@easymodo The bug is gone! Thanks :)

**Comment by kattjevfel** [comment]
Just tested a1058ef and it works with latest mpv!

**Comment by pviotti** [comment]
it's great that it's been fixed. I suggest releasing a new version with this fix (0.91 was released ~1 year ago, and I see many improvements have been made in the meantime). Thanks

**Comment by easymodo** [comment]
@pviotti Release done. Sorry for dragging this out.
